nautobot/extras/views.py

```diff
@@ -1523,6 +1523,18 @@ def extra_post_save_action(self, obj, form):
             obj.job_queues_override = True
             obj.job_queues.set(cleaned_data["job_queues"])
 
+        # Handle default job queue
+        clear_override_field = "clear_default_job_queue_override"
+        reset_override = cleaned_data.get(clear_override_field, False)
+        if reset_override:
+            meta_task_queues = obj.job_class.task_queues
+            job_queues = []
+            obj.default_job_queue_override = False
+            obj.default_job_queue = JobQueue.objects.get(name=meta_task_queues[0]) if meta_task_queues else None
```

Comment:
```suggestion
            obj.default_job_queue = JobQueue.objects.get(name=meta_task_queues[0] if meta_task_queues else settings.CELERY_TASK_DEFAULT_QUEUE)
```

Reply:
Might actually need `get_or_create` here to create the default queue if it doesn't exist?